codestodians/food-basket-front-end
# [Bug: #151350439]

#### What does this PR do?
- Clears the errors in the browser console arising from illegal nesting.

#### Description of Task to be completed?
- Disable navigation from all header(Navbar) menu items until a decision is made on the items to be included.
- Preserve the existing names of menu items already included for an aesthetic purpose.

#### How should this be manually tested?
- Start the backend and frontend servers.
- Open the browser console.
- It should be free from warnings and errors.

#### What are the relevant pivotal tracker stories?
- [#151350439](https://www.pivotaltracker.com/story/show/151350439)
